index.html

```diff
@@ -0,0 +1,380 @@
+				</li>
+				<li>
+					<a href="" class="media-link"><img src="./images/githubIcon.svg" alt="githubIcon"
+							class="media-icon"></a>
+				</li>
+				<li>
+					<a href="" class="media-link"><img src="./images/inconito.svg" alt="inconitoIcon"
+							class="media-icon"></a>
+				</li>
+			</ul>
+		</section>
+
+		<!-- Project section-->
+
+		<section id="section-project">
+			<div class="details-card">
+				<div class="project-image1"></div>
+				<div class="card-info">
+					<div class="project-description">
+						<div class="project-details">
+							<h1 class="project-title">Tonic</h1>
+							<div class="client-info">
+								<p class="canopy">CANOPY</p>
+								<ul class="Cinf">
+									<li>Back End Dev</li>
+									<li>2015</li>
+								</ul>
+							</div>
+						</div>
+						<div class="project-info">
+							<p>
+								A daily selection of privately personalized reads;
+								no accounts or sign-ups required.
+							</p>
+						</div>
+						<div class="project-tag">
+							<ul class="pl">
+								<li>html</li>
+								<li>css</li>
+								<li>javascript</li>
+							</ul>
+						</div>
+					</div>
+					<div class="btn-project">
+						<a href="">See Project</a>
+					</div>
+				</div>
+			</div>
+			<div class="details-card details2">
+				<div class="project-image2"></div>
+				<div class="card-info">
+					<div class="project-description">
+						<div class="project-details">
+							<h1 class="project-title">Multi-Post Stories</h1>
+							<div class="client-info">
+								<p class="canopy">CANOPY</p>
+								<ul class="Cinf">
+									<li>Back End Dev</li>
+									<li>2015</li>
+								</ul>
+							</div>
+
+						</div>
+						<div class="project-info">
+							<p>
+								A daily selection of privately personalized reads;
+								no accounts or sign-ups required.
+							</p>
+						</div>
+						<div class="project-tag">
+							<ul class="pl">
+								<li>html</li>
+								<li>css</li>
+								<li>javascript</li>
+							</ul>
+						</div>
+					</div>
+					<div class="btn-project">
+						<a href="">See Project</a>
+					</div>
+
+				</div>
+			</div>
+			<div class="details-card">
+				<div class="project-image3"></div>
+				<div class="card-info">
+					<div class="project-description">
+						<div class="project-details">
+							<h1 class="project-title">Tonic</h1>
+							<div class="client-info">
+								<p class="canopy">CANOPY</p>
+								<ul class="Cinf">
+									<li>Back End Dev</li>
+									<li>2015</li>
+								</ul>
+							</div>
+						</div>
+						<div class="project-info">
+							<p>
+								A daily selection of privately personalized reads;
+								no accounts or sign-ups required.
+							</p>
+						</div>
+						<div class="project-tag">
+							<ul class="pl">
+								<li>html</li>
+								<li>css</li>
+								<li>javascript</li>
+							</ul>
+						</div>
+					</div>
+					<div class="btn-project">
+						<a href="">See Project</a>
```

Comment:
- [ ] Please ensure that the right hover effects are applied to your button. It currently looks different from the Figma design. :100: 
- Your design
![image](https://user-images.githubusercontent.com/102699958/220822283-ee7d4ff6-4ba5-406f-8359-2037496c72a7.png)
- The Figma Design
![image](https://user-images.githubusercontent.com/102699958/220822337-d97810de-dcfa-4015-a2be-607273036df3.png)